packages/react-components/src/templates/AnalyticsLeadershipPageBody.tsx

```diff
@@ -119,11 +127,18 @@ const LeadershipPageBody: React.FC<LeadershipAndMembershipAnalyticsProps> = ({
     </div>
     <div css={searchContainerStyles}>
       <Subtitle>Teams:</Subtitle>
-      <span css={searchStyles}>
-        <SearchField
-          value={searchQuery}
+      <span role="search" css={searchStyles}>
+        <MultiSelect
```

Comment:
There's a 15px marginTop in this component that makes "Teams" to seems unaligned. Maybe we could add noMargin props as there is in another components :)
<img width="985" alt="Screenshot 2024-05-29 at 09 24 51" src="https://github.com/yldio/asap-hub/assets/16595804/b76ef800-6929-4f90-b766-d8b653914179">